src/shared/netif-naming-scheme.h

```diff
@@ -46,6 +46,8 @@ typedef enum NamingSchemeFlags {
         NAMING_V247 = NAMING_V245 | NAMING_BRIDGE_NO_SLOT,
         NAMING_V249 = NAMING_V247 | NAMING_SLOT_FUNCTION_ID | NAMING_16BIT_INDEX | NAMING_REPLACE_STRICTLY,
 
+        EXTRA_NET_NAMING_SCHEMES
```

Comment:
you don't need this here, right?

I mean, it would totally suffice to extend naming_schemes[] with the bit masks. Having them in the enum here doesn't really help anyone, since w never want to access them via their enum name anywhere.

Reply:
True. I'll drop this.

Reply:
Ah, no. We use the enum in two places: once in the test, but also potentially in the definition of `DEFAULT_NET_NAMING_SCHEME`. So I think it's better to keep this.

Reply:
i see